pkg/cmd/safe/commands.go

```diff
@@ -39,27 +60,30 @@ func isDryRun(cmd []string) bool {
 }
 
 func getSafeCommands() (*Commands, error) {
-	if commands := os.Getenv(KubectlSafeCommands); commands != "" {
-		return paresSafeCommands(commands)
+	commands, err := parseCommands(KubectlSafeCommands)
+	if err != nil {
+		return nil, err
 	}
-	return &DefaultSafeCommands, nil
+	if len(commands.cmds) == 0 {
+		return &DefaultSafeCommands, nil
+	}
+	return commands, nil
 }
 
-func paresSafeCommands(commands string) (*Commands, error) {
+func parseCommands(env string) (*Commands, error) {
+	commands := os.Getenv(env)
+	if commands == "" {
+		return &EmptyCommands, nil
+	}
 	// commands can be a csv of strings or a file path
-	_, err := os.Stat(commands)
-	switch {
-	case os.IsNotExist(err):
+	if _, err := os.Stat(commands); errors.Is(err, fs.ErrNotExist) {
```

Comment:
I wonder if having a separate ENV variable for the file will be more explicit than checking for file first and then falling back on comma separated args. Note this doesn't break my usage, just my 2c, thinking more about less corner cases to support and worry about

Reply:
definitely something to think about. My thought was to avoid env overload. Maybe should use something like viper